h5py/__init__.py

```diff
@@ -56,7 +56,7 @@
 from . import h5a, h5d, h5ds, h5f, h5fd, h5g, h5r, h5s, h5t, h5p, h5z, h5pl
 
 from ._hl import filters
-from ._hl.base import is_hdf5, HLObject, Empty
+from ._hl.base import is_hdf5, HLObject, Empty, dlapl, dlcpl
```

Comment:
```suggestion
from ._hl.base import is_hdf5, HLObject, Empty
```

Modifying global default property lists is definitely not how we should encourage people to use the API.

We should instead come up with some way to get custom options to the operations that need them, whether by passing in a property list or passing in the relevant options and constructing a property list internally. This may be somewhat more complex, so we might want to expose the low-level functionality in this PR and then figure out the high-level API as a later step.

Reply:
reverted